### PR TITLE
contractcourt: homogenize naming convention

### PR DESCRIPTION
**Pull request by ProofOfKeags**
## Change Description
This is a strict refactor and document cleanup

## Steps to Test
N/A

### Testing
- [ ] Your PR passes all CI checks.
- [x] Tests covering the positive and negative (error paths) are included.
- [x] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [x] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [x] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [x] Any new logging statements use an appropriate subsystem and logging level.
- [x] Any new lncli commands have appropriate tags in the comments for the rpc in the proto file.
- [x]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

File: contractcourt/breacharbiter.go [hunk 1]
```diff
@@ -171,15 +173,15 @@ type BreachConfig struct {
...
+// BreachArbitrator is a special subsystem which is responsible for watching and
 // acting on the detection of any attempted uncooperative channel breaches by
 // channel counterparties. This file essentially acts as deterrence code for
 // those attempting to launch attacks against the daemon. In practice it's
 // expected that the logic in this file never gets executed, but it is
 // important to have it in place just in case we encounter cheating channel
 // counterparties.
 // TODO(roasbeef): closures in config for subsystem pointers to decouple?
-type BreachArbiter struct {
+type BreachArbitrator struct {
```
Comment (yyforyongyu) [inline contractcourt/breacharbiter.go:184]: 🌹

**Review by yyforyongyu — APPROVED** [review]
Thanks for cleaning this up!

File: contractcourt/breach_arbitrator.go [hunk 2]
```diff
@@ -684,8 +686,11 @@ func updateBreachInfo(breachInfo *retributionInfo, spends []spend) (
 // the lingering funds within the channel into the daemon's wallet.
 //
 // NOTE: This MUST be run as a goroutine.
-func (b *BreachArbiter) exactRetribution(confChan *chainntnfs.ConfirmationEvent,
-	breachInfo *retributionInfo) {
+//
+//nolint:funlen
+func (b *BreachArbitrator) exactRetribution(
+	confChan *chainntnfs.ConfirmationEvent, breachInfo *retributionInfo,
+) {
```
Comment (guggero) [inline contractcourt/breach_arbitrator.go:693]: Why are we inventing new ways to format our function declarations? Can we please stick with the ones defined [here](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md) or update the doc and have a discussion about it in the PR that updates it?

Comment (ProofOfKeags) [reply contractcourt/breach_arbitrator.go:693]: Because I'm still learning them (they aren't intuitive) and the linter didn't catch it. Honest mistake.

Comment (guggero) [reply contractcourt/breach_arbitrator.go]: Fair enough. I guess it didn't help that we [suddenly started using a new format with the Taproot Channel PRs](https://github.com/lightningnetwork/lnd/blob/ce93b236aacdcb3f810c29fff3833228d74dea3c/input/musig2.go#L444) that IMO is [explicitly documented as being wrong in the formatting guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#wrapping-long-function-definitions). Still trying to convince @Roasbeef to roll that back...

File: contractcourt/breacharbiter.go [hunk 3]
```diff
@@ -943,15 +947,18 @@ func (b *BreachArbiter) cleanupBreach(chanPoint *wire.OutPoint) error {
...
+// BreachArbitrator will attempt to sweep ALL funds within the channel using the
 // information provided within the BreachRetribution generated due to the
 // breach of channel contract. The funds will be swept only after the breaching
 // transaction receives a necessary number of confirmations.
 //
 // NOTE: This MUST be run as a goroutine.
-func (b *BreachArbiter) handleBreachHandoff(breachEvent *ContractBreachEvent) {
+func (b *BreachArbitrator) handleBreachHandoff(
+	breachEvent *ContractBreachEvent,
+) {
```
Comment (guggero) [inline contractcourt/breacharbiter.go:960]: nit: closing parenthesis should be on previous line.

**Review by guggero — APPROVED** [review]
Just one nit, otherwise LGTM :tada:

**Pull request merged**